src/utils/api/peers.js

```diff
@@ -1,12 +1,16 @@
+import { loadingStarted, loadingFinished } from '../../utils/loading';
+
 /* eslint-disable  */
 export const requestToActivePeer = (activePeer, path, urlParams) =>
   new Promise((resolve, reject) => {
+    loadingStarted(path);
```

Comment:
I think we should use loading actions inside our component instead of using them inside our api utils